website/docs/API.md

```diff
@@ -223,6 +250,24 @@ Hold the value of latest render call for easier access to query and other functi
 
 Its value is automatically cleared after each test by calling [`cleanup`](#cleanup). If no `render` call has been made in a given test then it holds a special object that implements `RenderResult` but throws a helpful error on each property and method access.
 
+This can also be used to build test utils that would normally require to be in render scope, either in a test file or globally for your project. For instance in a test file where we fill a form in multiple tests: 
```

Comment:
The concept of building test utils seems like a useful idea, however the `typeEmail` example seems quite artificial and I would not recommend using such util over explicit `fireEvent` + query.

@pierrezimmermannbam do you have idea for something that we could actually recommend to your users here?

Reply:
This is something is sometimes do when I test forms and have a lot of tests, imo it makes tests more readable and very easy to understand while not making them harder to debug since you still have the same error messages. It's even more interesting when it allows to hide testID usage since the queries by testID do not really help to understand the test. An example of this is waiting for a loader to disapear, which is often done in every test when testing at page level

```tsx
const waitForLoaderToDisapear = async () => {
   await waitForElementToBeRemoved(() => screen.getByTestID('PageLoader'));
}
```

Does this example make more sense to you ?